src/esp/gfx/WindowlessContext.cpp

```diff
@@ -18,7 +18,7 @@
 #include <fcntl.h>
 #include <sys/stat.h>
 #include <unistd.h>
-#elif __win32__
+#elif __win32__ || defined(_WIN32)
```

Comment:
@msavva @erikwijmans we could use `CORRADE_TARGET_{APPLE,UNIX,WINDOWS}` here for consistency I think?

Reply:
@mosra that sounds like a good idea

Reply:
Seconded!

Reply:
Done as a part of #106.